dev/postgres/fixtures/fixture-1.sql

```diff
@@ -4,3 +4,5 @@ create table users (
 );
 
 alter table users replica identity full;
+
+create publication events for all tables;
```

Comment:
this should be configurable

Reply:
This is just a fixute, I've added support for passing publication name in the environment